tests/test_git.py

```diff
@@ -146,6 +146,84 @@ def test_fetch_since_date(self):
 
         shutil.rmtree(new_path)
 
+    def test_fetch_branch(self):
+        """Test whether commits are fetched from a Git repository for a given branch"""
+
+        new_path = os.path.join(self.tmp_path, 'newgit')
+
+        import logging
```

Comment:
Oooops. Sorry. That's the rests of a test that I was doing to check what happened in some circunstances...